rskj-core/src/test/java/co/rsk/peg/PegoutCreationEntryTest.java

```diff
@@ -0,0 +1,65 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.Sha256Hash;
+import co.rsk.crypto.Keccak256;
+import junit.framework.TestCase;
+
+public class PegoutCreationEntryTest extends TestCase {
```

Comment:
Why does it extend from TestCase?

Reply:
For no reason in particular. I just generated the test class using the IDE helper generator. I can modify the test class to be like the existing ones.